src/pancam.py

```diff
@@ -0,0 +1,22 @@
+from follower import Follower
+
+follower = Follower()
+
+def seek():
+    #tag_data = follower.detect()
+    turn = follower.turn()
```

Comment:
`follower.turn()` turns the wheels to `_turn_angle`. `_turn_angle` isn't set until `follower.detect()`.